sources/dfmc/reader/tests/literal-test-suite.dylan

```diff
@@ -17,9 +17,11 @@ define function verify-presentation
 end function verify-presentation;
 
 define test binary-integer-literal-test ()
-  let f = read-fragment("#b010");
-  verify-literal(f, 2, <integer-fragment>);
-
+  verify-literal(read-fragment("#b10"), 2, <integer-fragment>);
+  verify-literal(read-fragment("#b010"), 2, <integer-fragment>);
+  verify-literal(read-fragment("#b1111_0000"), 240, <integer-fragment>);
+  assert-false(read-fragment("#b10_"));
+  assert-signals(<invalid-token>, read-fragment("#b_10"));
```

Comment:
Should also test with multiple underscores.

Reply:
Done.